docs/source/yelpsoa_configs.rst

```diff
@@ -39,6 +39,10 @@ instance MAY have:
     left on device' errors if they attempt to exceed these limits, and then be
     unable to write any more data to disk.
 
+  * ``ulimit``: Dictionary of ulimit options that are passed to Docker. Defaults
```

Comment:
Can you add an example? 

Also reading the upstream docs it looks like it gets taken in as an array?
https://docs.docker.com/engine/reference/commandline/run/#/set-ulimits-in-container-ulimit